test/e2e-go/features/transactions/transactionPool_test.go

```diff
@@ -27,6 +27,7 @@ import (
 )
 
 func TestTransactionPoolOrderingAndClearing(t *testing.T) {
```

Comment:
If there is not a Jira bug for this yet, please create one.  Ideally we should reference the jira task in the Skip message.  Bonus points for that, but i'll approve and leave it to you.

Reply:
It's on the internal jira as GOAL2-779